exercises/grains/src/test/scala/GrainsTest.scala

```diff
@@ -1,52 +1,59 @@
-import org.scalatest._
+import org.scalatest.{Matchers, FunSuite}
 
+/** @version 1.0.0 */
 class GrainsTest extends FunSuite with Matchers {
-  test ("square 1") {
+
+  test("1") {
     Grains.square(1) should be (Some(1))
   }
 
-  test ("square 2") {
+  test("2") {
     pending
     Grains.square(2) should be (Some(2))
   }
 
-  test ("square 3") {
+  test("3") {
     pending
     Grains.square(3) should be (Some(4))
   }
 
-  test ("square 4") {
+  test("4") {
     pending
     Grains.square(4) should be (Some(8))
   }
 
-  test ("square 16") {
+  test("16") {
     pending
     Grains.square(16) should be (Some(32768))
   }
 
-  test ("square 32") {
-    Grains.square(32) should be (Some(2147483648L))
+  test("32") {
+    pending
+    Grains.square(32) should be (Some(BigInt("2147483648")))
   }
 
-  test ("square 64") {
+  test("64") {
     pending
     Grains.square(64) should be (Some(BigInt("9223372036854775808")))
   }
 
-  test ("square negative") {
+  test("square 0 raises an exception") {
+    pending
+    Grains.square(0) should be (None)
+  }
+
+  test("negative square raises an exception") {
     pending
     Grains.square(-1) should be (None)
   }
 
-  test ("square bigger than 64") {
+  test("square greater than 64 raises an exception") {
     pending
     Grains.square(65) should be (None)
   }
 
-  test ("total grains") {
+  test("returns the total number of grains on the board") {
     pending
     Grains.total should be (BigInt("18446744073709551615"))
```

Comment:
When I run the generator locally I get an `IllegalStateException` in L21 instead?!